packages/suite/src/views/wallet/coinmarket/common/CoinmarketLayout/CoinmarketAccountTransactions/CoinmarketAccountTransactions.tsx

```diff
@@ -84,7 +84,7 @@ export const CoinmarketAccountTransactions = () => {
                                 {buyTransactions.length} <Translation id="TR_TRADE_BUYS" /> •{' '}
                                 {sellTransactions.length} <Translation id="TR_TRADE_SELLS" /> •{' '}
                                 {exchangeTransactions.length}{' '}
-                                <Translation id="TR_TRADE_EXCHANGES" />
+                                <Translation id="TR_COINMARKET_SWAPS" />
```

Comment:
How about creating one big translation string for the content of `TransactionCount` and just use values for numbers.
Or maybe better create translation strings together with numbers so that it works with plurals - 1 buy, 2 buys